securedrop/bin/upload-screenshots

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# A maintenance script for uploading all screenshots generated by the page
+# layout tests to Weblate, to provide context for translators. Only the
+# English language screenshots are uploaded.
+#
+# Currently this script does not perform checks for existing screenshots;
+# it just blindly uploads the entire set of screenshots, which then have
+# to be manually associated with the correct strings.
+
+# Standard shell safety settings
+set -e
+set -u
+set -o pipefail
+
+if [ -z "${WEBLATE_BASE_URL:-}" ]; then
+   WEBLATE_BASE_URL="https://weblate.securedrop.org/"
+fi
+
+if [ -z "${WEBLATE_API_KEY:-}" ]; then
+   echo "WEBLATE_API_KEY environment variable is not set. This script requires"
+   echo "an API key to be configured to perform file uploads. You can obtain"
+   echo "one via this link: ${WEBLATE_BASE_URL}accounts/profile/#api"
+   exit 1
+fi
+
+BASEDIR=$(dirname "$0")
```

Comment:
This works fine, but [elsewhere](https://github.com/freedomofpress/securedrop/blob/96debdc98be53bafac89b5ddaea761a0f71894fa/securedrop/bin/dev-shell#L11) we resolve the repo root with `TOPLEVEL=$(git rev-parse --show-toplevel)`. It's a little simpler and more robust than relative paths, should the script ever be moved. (I know, probably won't. :roll_eyes:)